junit/README.md

```diff
@@ -0,0 +1,50 @@
+Cucumber JUnit 
+==============
+
+Use JUnit to execute cucumber scenarios.
+
+Add the `cucumber-junit` dependency to your pom.
+
+```xml
+<dependencies>
+  [...]
+    <dependency>
+        <groupId>io.cucumber</groupId>
+        <artifactId>cucumber-junit</artifactId>
+        <version>${cucumber.version}</version>
+        <scope>test</scope>
+    </dependency>
+  [...]
+</dependencies>
+```
+
+Create an empty class that uses the Cucumber JUnit runner.
+
+```java
+package com.example;
+
+import cucumber.api.CucumberOptions;
+import cucumber.api.junit.Cucumber;
+import org.junit.runner.RunWith;
+
+@RunWith(Cucumber.class)
+@CucumberOptions(plugin = "json:target/cucumber-report.json")
+public class RunCukesTest {
+}
+```
+
+This will execute all scenarios in the package of the runner. By default glue code is assumed to be in the same 
+package. The `@CucumberOptions` can be used to provide additional configuration to the runner.
+
+## Using JUnit Rules ##
+
+Cucumber supports JUnits `@ClassRule`, `@BeforeClass` and `@AfterClass` annotations. These will be invoked around the 
```

Comment:
This paragraph might not be clear Cucumber users who are not full-on software developers (yet). 

Particularly:

"Cucumber supports JUnits `@ClassRule`, `@BeforeClass` and `@AfterClass` annotations. These will be invoked around the 
+suite of features. " => what does it mean to run 'around the suite of features'? Is this like a BeforeAll/AfterAll?

"Using these is not recommended as it limits the portability between different runners. " => what is 'portability between different runners'? Why is limiting this a bad thing?

"Instead it is
+recommended to use Cucumbers `Before` and `After` hooks to setup scaffolding." => How to use 'Before' and 'After' hooks? What is 'setup scaffolding'?

Maybe this is not the place to answer all of these questions, but I think this could be clarified a little to be clearer to a wider audience.

Reply:
Do we have some documentation about the Before and After hooks? I am assuming people are familiar with cucumber when they read this.

Reply:
Yes, we do. Currently here: https://cucumber.netlify.com/hooks/